packages/iov-ethereum/src/utils.ts

```diff
@@ -73,3 +75,25 @@ export function fromBcpChainId(chainId: ChainId): number {
 
   return Uint53.fromString(rest).toNumber();
 }
+
+/**
+ * A function to determine if a transaction is interpreted as ERC20 transfer.
+ * We can not know for sure if it was a ERC20 call without knowledge of the recipient type,
+ * which is not available at the codec level.
+ */
+export function shouldBeInterpretedAsErc20Transfer(input: Uint8Array, ethQuantity: string): boolean {
+  if (ethQuantity !== "0") {
```

Comment:
Is this definitely required?

Reply:
The whole guessing method is pointless since we have ERC20 contract addresses available in the codec now. Will cleanup after merge to get this in quicker